src/Styling/TrackList.css

```diff
@@ -0,0 +1,3 @@
+.TrackList {
+    width: 100%;
```

Comment:
does this style class even do anything? In the TrackList.js file there is only 1 div, and if I’m not mistaken he default width value is already 100%